src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstSide;
+    private int secondSide;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstSide, int secondSide, int height, Color color) {

```

Comment:
It would be better to check if the given sides and height are positive values. Negative or zero values are not valid for these parameters.